feat(core): Add rollup-plugin-svg

ryanirilli opened the pull request:
Adds the [rollup-plugin-image](https://github.com/rollup/rollup-plugin-image)

ryanirilli commented:
Ah, my bad! fixed

evocateur commented:
I think rollup-plugin-image might be incompatible with recent versions of Rollup? https://github.com/rollup/rollup-plugin-image/issues/2 is a bit alarming, in any case...

Have you confirmed that this works locally? It'd be really swell if there was a test of this functionality...

ryanirilli commented:
I updated the PR to use [rollup-plugin-svg](https://github.com/antony/rollup-plugin-svg) instead and added an svg import to the integration test to confirm the usage of the loader

evocateur commented on package-lock.json:
```diff
@@ -8807,6 +8808,14 @@
     },
+    "rollup-plugin-svg": {
+      "version": "1.0.1",
+      "resolved": "http://znpm.in.zillow.net:31050/r/rollup-plugin-svg/_attachments/rollup-plugin-svg-1.0.1.tgz",
```
sorry, you need to use `--registry https://registry.npmjs.org/` when calling npm install, people on the internet dont' have access to Zillow's private network 😅

evocateur commented on packages/pectin-core/test/pectin-core.test.js:
```diff
@@ -175,7 +176,10 @@ describe('pectin-core', () => {
-                'index.js': File('export default function foo() {};'),
+                'test.svg': File(`test`),
+                'index.js': File(
+                    `import svgTest from './test.svg'; export default function foo() {};`
```
I think this needs to use the `svgTest` import, otherwise it gets stripped away. Like, return it from `foo()` or something. The snapshot should be changing, basically.
